### ISSUE TITLE
Add a short project description (readme.txt)

### ISSUE THREAD

**Issue by aratiu** [MariusRA/Office-Time-Tracker#1]

**Comment by aratiu** [comment]
The normal workflow is to leave task open until the pull-request get approved and merged into master branch.
You should better make usage of "Linked pull request" or leave a comment with a link corresponding pull-request.

**Comment by aratiu** [comment]
Please work on branches, don't push directly on master! 
As a general approach over the tasks, you should:
1) Open the task and refine description if is unclear, or leave a comment.
2) When is clear, create a branch:
> git checkout master
> git pull origin
> git checkout -b <branch_name>
3) Make sure you are on the apropiate  branch when you start modifying otherwise switch with:
> git checkout <branch_name>
4) Make commit(s)
5) Publish the branch:
> git push
6) Once the branch is ready create the pull-request and link it to the task.

**Comment by aratiu** [comment]
For the current task, description you've already added to master is fine but you should rename the file to lowercase as we agreed (i.e. readme.txt):
> git checkout master
> git pull origin
> git checkout -b project_scope
> git mv README.txt readme.txt
> git commit -m "Rename readme file"
> git push

Also add the tool set (hint: Android Studio, git, gihub - for repository and task management, test device - Nexus 5 with Android 6.0 Marshmallow, etc.) 
> git checkout project_scope
> git pull origin
> change readme content
> git add readme.txt
> git commit -m "Update project scope"
> git push

If you want to make corrections:
> edit file
> git add readme.txt
> git commit --amend 
> git push

When ready, please make the pull request, link it to the task, but leave it open.

NOTE: please contact me directly whenever you have a git error which you cannot handle yourself.

**Comment by aratiu** [comment]
Pull-request approved and merged. I'm going to close the issue.
